ghpages.mk

```diff
@@ -68,9 +71,21 @@ $(GHPAGES_TMP): fetch-ghpages
 
 PUBLISHED := index.html $(drafts_html) $(drafts_txt)
 
+ifeq (master,$(SOURCE_BRANCH))
+ifeq (true,$(PUSH_GHPAGES))
+EXTANT_BRANCHES := $(subst refs/heads/,,$(foreach remote,$(shell git remote),$(shell git ls-remote --heads $(remote) | cut -f 2)))
```

Comment:
The difference, I think, is that ls-remote pulls live data from the remotes, while yours is only current as of the last fetch.  Also, remote-tracking branches aren't cleaned up automatically.

Though I could fetch all remotes and prune all remotes first, I was trying to avoid side-effects of gathering the data.